client/signup/jetpack-connect/live-chat-button.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import olarkActions from 'lib/olark-store/actions';
+
+export default React.createClass( {
+	displayName: 'JetpackConnectLiveChatButton',
+
+	handleClick() {
+		olarkActions.expandBox();
+	},
+
+	render() {
+		if ( ! this.props.userModule.get() ) {
```

Comment:
I'm just noticing that plans.jsx does not pass down `userModule` in props.
I propose we import `user` directly into this component, allowing this component to function independent of its parent.